.readthedocs.yaml

```diff
@@ -0,0 +1,31 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build all formats
+formats: all
+
+python:
+  # Install our python package before building the docs
+  install:
+    - requirements: docs/source/requirements.txt
+    - method: pip
+      path: .
+      
+ # Set the version of Python and other tools you might need
+build:
+  os: "ubuntu-20.04"
+  tools:
+    python: "3.8"
+
+
+# Build documentation in the docs/source/ directory with Sphinx
+sphinx:
+  builder: html
+  configuration: docs/source/conf.py
+  fail_on_warning: false
```

Comment:
I have set it to false for the time being as I am getting an warning which is stopping the build process .
I tried resolving the error , but I am unable figure it out.
### warning :
>/home/docs/checkouts/readthedocs.org/user_builds/greedybear-docs/checkouts/latest/docs/source/OpenAPI.rst:44: WARNING: Definition list ends without a blank line; unexpected unindent.

Can u help me out?

Reply:
well they are telling you which is the problem. See below